content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -158,6 +158,21 @@ spec:
 
 ### Caveats and limitations when preserving source IPs
 
+Health check node port serves HTTP traffic. The response body contains JSON service name, namespace and number 
```

Comment:
This doesn't belong under that heading.

I'm not sure it really belongs in this document at all though; this is information that LoadBalancer implementations need to know. It's not really useful to end users of Kubernetes...

Reply:
@sftim we want to add this information about the healthceck Nodeport used for Services with type LoadBalancer when `externalTrafficPolicy` is set to Local , so implementations can now the nodes with pods and how many pods are running on the node. Since you were restructuring this documentation recently, can you help us to identify the best place to add it?

Reply:
I'd update https://kubernetes.io/docs/reference/networking/virtual-ips/

Reply:
If it helps, you can frame it as a note along the lines of “_Older Kubernetes releases had a shortcoming where &hellip;_”